manifests/maistra.io_servicemeshcontrolplanes.yaml

```diff
@@ -2730,6 +2730,7 @@ spec:
                               format: int32
                               type: integer
                             protocol:
+                              default: TCP
```

Comment:
I think we previously explicitly removed this from the CRDs due to some k8s bug. Is that resolved with v1?

Reply:
Good catch. I don't remember.  I'll need to amend the changes for maistra/istio-operator, as I did not update the operator CRDs to v1.

Reply:
yeah, it's fixed in v1.